### ISSUE TITLE
Broken download URLs on lakka.tv site

### ISSUE THREAD

**Issue by sgskinner** [libretro/Lakka-LibreELEC#2016]
- Which version of Lakka are you using?
    -  Attempting to install Lakka-x11.x86_64-5.0
- What system hardware are you using?
    - intel 13900k, nvidia 4080
- What did you do?
   - Attempted to download image as currently linked on site:
   - https://le-builds.lakka.tv/x11.x86_64/Lakka-x11.x86_64-5.0.img.gz
   - ![broken-url](https://github.com/user-attachments/assets/39b64363-9554-4dca-95b1-0053f5b00c5d)
- What did you expect to happen?
    - Download linked image file
- What happened instead?
    - Cloudflare timeout, host error 522

The issue is a bad URL, `le-builds` should be `le.builds` (i.e. hyphen should be period), corrected URL works as expected:
- https://le.builds.lakka.tv/x11.x86_64/Lakka-x11.x86_64-5.0.img.gz

Perusing the other image links, it appears to be an issue for all non-dev/non-nightly files.

**Comment by jph76** [comment]
@sgskinner thank you for saving my day!

Can confirm this fixes download for Raspberry Pi 5 image.

**Comment by faywong** [comment]
> Reference

Yes, it fixes well. Many many thx to you. Also expect to be fixed in upstream.

https://le.builds.lakka.tv/RPi4.aarch64/Lakka-RPi4.aarch64-5.0.img.gz is good.
https://le-builds.lakka.tv/RPi4.aarch64/Lakka-RPi4.aarch64-5.0.img.gz is broken.
